<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>1012</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="message_TextBrowser">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>588</y>
      <width>421</width>
      <height>331</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="client_table">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>378</y>
      <width>601</width>
      <height>541</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>이름</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>집중도</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>기타</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>기타2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>기타3</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="Concentration_box">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>78</y>
      <width>621</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Concentration Graph</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>601</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="Graph_layout"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ClientInfo_box">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>348</y>
      <width>621</width>
      <height>591</height>
     </rect>
    </property>
    <property name="title">
     <string>Client Info</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Message_box">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>548</y>
      <width>441</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Message</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="UserCam_box">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>78</y>
      <width>441</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>Cam</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="Input_cam">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>106</y>
      <width>421</width>
      <height>411</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>419</width>
       <height>409</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>IP</string>
    </property>
    <widget class="QLabel" name="IP_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>UserCam_box</zorder>
   <zorder>ClientInfo_box</zorder>
   <zorder>Message_box</zorder>
   <zorder>Input_cam</zorder>
   <zorder>Concentration_box</zorder>
   <zorder>message_TextBrowser</zorder>
   <zorder>client_table</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1179</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
